db/flush_job_test.cc

```diff
@@ -165,6 +168,72 @@ TEST_F(FlushJobTest, NonEmpty) {
   job_context.Clean();
 }
 
+TEST_F(FlushJobTest, FlushSomeMemTables) {
```

Comment:
minor: When I read the test's name, I expected it is going to flush a subset of the memtables. Since it flushes all memtables, perhaps we can call it "FlushAllMemTables", or "FlushMultipleMemTables".